src/content/plugins/split-chunks-plugin.md

```diff
@@ -181,7 +182,7 @@ splitChunks: {
 			test: /[\\/]node_modules[\\/]/,
 			priority: -10
 		},
-    default: {
```

Comment:
i think this was fixed somewhere in an open PR before.

Reply:
I'm not sure what you want me to do about that